chainio/clients/fireblocks/client.go

```diff
@@ -0,0 +1,162 @@
+package fireblocks
+
+import (
+	"bytes"
+	"context"
+	"crypto/rsa"
+	"crypto/sha256"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/Layr-Labs/eigensdk-go/logging"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/golang-jwt/jwt"
+	"github.com/google/uuid"
+)
+
+type FireblocksTxID string
+
+type FireblocksClient interface {
+	ContractCall(ctx context.Context, body *ContractCallRequest) (*ContractCallResponse, error)
```

Comment:
Is this really a contractCall (just reading data)? Or is it a send transaction?

I'm guessing its actually signing and sending a tx as that's the only goal of fireblocks right? I see that fireblocks itself has named this operator CONTRACT_CALL in their api, any idea why? It's confusing.

Reply:
This is so weird. See for eg https://ethereum.stackexchange.com/questions/765/what-is-the-difference-between-a-transaction-and-a-call
Can we at least document this as a comment above the method?

Reply:
Yeah, we had to ask a few clarifying questions to learn the behavior for this method. I'll add a comment